tests/cross/src/test/scala/tests/pc/AutoImplementAbstractMembersSuite.scala

```diff
@@ -326,7 +360,7 @@ class AutoImplementAbstractMembersSuite extends BaseCodeActionSuite {
   )
 
   checkEdit(
-    "jutil",
+    "jutil".tag(IgnoreScala3),
```

Comment:
What is the issue here? Could add a compat section instead?

Reply:
This is because we don't accept rename config to OverrideCompletion at this moment, we can fix this later, in another PR :+1:

Anyway, I'll add compact section for them.

Reply:
https://github.com/scalameta/metals/pull/3960/commits/b32d45959a2199e8ad7e5d3700ea3e3f681907ac